### ISSUE TITLE
Refuses to accept lobste.rs as a valid domain

### ISSUE THREAD

**Issue by singpolyma** [doits/Sparse-RSS#2]
https://lobste.rs/t/haskell.rss feed is not accepted to refresh, shows a Java exception about the URI

**Comment by Rushi98** [comment]
@singpolyma, The google code archive [page](https://code.google.com/archive/p/sparserss/) says the development has stopped. If you are interested, maybe create a fork and link it here.

**Comment by doits** [comment]
yep, since I don't use it anymore I am not developing it further, either. Feel free to fork it.
